ssh_automated_tests/core/connection.py

```diff
@@ -10,13 +10,17 @@ class Connection(ABC):
     def connect(self):
         pass
 
+    @abstractmethod
+    def disconnect(self):
+        pass
 
-class SshConnection:
+
+class SshConnection(Connection):
```

Comment:
In this realization `SshConnection` is a specific case of connection that have a possibility to open shell or sftp. Like it is in real. Is it bad to have such object?
If it's not ok, so it has to be separate implementation of `Connection` or I can just inherit `SshConnection` by `ShellConnection` and add some functionality? Because they'll have pretty similar structure except one additional method.